int8 is_ready
---
bool success
